src/pages/Groups/fines/FineItem.tsx

```diff
@@ -117,22 +113,22 @@ const FineItem = ({ fine, groupSlug, isAdmin, hideUserInfo, ...props }: FineItem
             <>
               <Stack direction={{ xs: 'column', md: 'row' }} gap={1}>
                 <Button
-                  color={fine.approved ? 'success' : 'error'}
+                  color={fine.approved ? 'error' : 'success'}
                   disabled={fine.payed}
                   fullWidth
                   onClick={toggleApproved}
                   startIcon={<ApprovedIcon />}
-                  variant='contained'>
-                  {fine.approved ? 'Godkjent' : 'Ikke godkjent'}
+                  variant='outlined'>
+                  {fine.approved ? 'Merk som ikke godkjent' : 'Merk som godkjent'}
```

Comment:
Synes det burde være contained slik at noen av knappene står mer ut enn andre. Da veileder vi bruker til hva som skal klikkes på
```suggestion
                  variant='contained'>
                  Merk som {fine.approved ? 'ikke godkjent' : 'godkjent'}
```

Reply:
Tanker? ^